diff -urN aa/gnome-pkgview-1.0.6/configure.in gnome-pkgview-1.0.6/configure.in
--- aa/gnome-pkgview-1.0.6/configure.in	2004-04-22 19:05:12.000000000 +0200
+++ gnome-pkgview-1.0.6/configure.in	2004-05-01 12:34:04.400129680 +0200
@@ -24,7 +24,7 @@
 
 AC_DEFINE_UNQUOTED(GNOME_ICONDIR, "${prefix}/share/pixmaps")
 
-ALL_LINGUAS="am az be bg ca cs da de el en_GB eo es fr he hr it ja ko lv ml ms nl no pa pl pt pt_BR ru sk sr sr@Latn sv uk vi zh_CN zh_TW"
+ALL_LINGUAS="am az be bg ca cs da de el en_GB eo es fr he hr it ja ko lv ml ms nb nl pa pl pt pt_BR ru sk sr sr@Latn sv uk vi zh_CN zh_TW"
 
 GETTEXT_PACKAGE=gnome-pkgview
 AC_SUBST(GETTEXT_PACKAGE)
